sap-s4hana-cloud/2.7/modules/ROOT/pages/sap-s4hana-cloud-connector-studio.adoc

```diff
@@ -91,9 +91,9 @@ image::sap-hana-global-config-wizard.png["Global configuration elements screen w
   The user name used to initialize the session
   ** *Password* +
   The password used to authenticate the user
++
   * *OAuth 2 Application Connection Provider* +
-
-  For OAuth 2.0 authentication, enter the following required information:
+  In OAuth 2.0 authentication, a TLS context that comes with a truststore can't be added because the SDK does not expose the TLS context afterward. For OAuth 2.0 authentication, enter the following required information:
```

Comment:
```suggestion
  In OAuth 2.0 authentication, a TLS context that comes with a truststore can't be added because the SDK doesn't expose the TLS context afterward. For OAuth 2.0 authentication, enter required information:
```